app/src/test/java/com/github/multimatum_team/multimatum/DateTimeExtractorTest.kt

```diff
@@ -37,7 +37,7 @@ class DateTimeExtractorTest {
     }
 
     @Test
-    fun `18h_is_parsed_correctly`(){
+    fun `18h_is_parsed_correctly`() {
```

Comment:
Be careful when applying `ctrl+alt+L` to files you don't modifiy, you may create lots of conflicts for a very small benefit!

Reply:
Yes I mistakenly formatted the entire project, I agree its a bad idea but by the time I realized that, I had already modified a bunch of (relevant) files.